pandas/core/generic.py

```diff
@@ -1665,8 +1665,9 @@ def to_json(self, path_or_buf=None, orient=None, date_format=None,
         Parameters
         ----------
         path_or_buf : the path or buffer to write the result string
```

Comment:
The "the path or buffer to write the result string" is actually the explanation, so should go on the next line. On this line comes the type description, you can take a look at eg DataFrame.to_csv how this is typically explained.

Reply:
Cheers